Sources/Basics/Sandbox.swift

```diff
@@ -82,6 +90,15 @@ fileprivate func macOSSandboxProfile(
     return contents
 }
 
+fileprivate extension String {
+    var quotedAsSubpathForSandboxProfile: String {
+        return "\"" + self
+            .replacingOccurrences(of: "\\", with: "\\\\")
+            .replacingOccurrences(of: "\"", with: "\\\"")
+            + "\""
+    }
+}
```

Comment:
should this be a private extension on AbsolutePath?

Reply:
That's a good point — I'll do that.

Longer-term there should be some more organized way of doing quoting / escaping for a variety of purposes.  We could also consider using custom interpolators for it, since it doesn't matter so much what the string originally was.

This is also not the most elegant implementation of this, but it works and is called infrequently enough that we can pend that until we do something more comprehensive.

But for here I'll change it to extend AbsolutePath.

Reply:
Changed.